cmd/generic-handlers.go

```diff
@@ -679,9 +679,8 @@ func (f bucketForwardingHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 
 	bucket, object := request2BucketObjectName(r)
 
-	// ListBucket requests should be handled at current endpoint as
-	// all buckets data can be fetched from here.
-	if r.Method == http.MethodGet && bucket == "" && object == "" {
+	// No bucket requests should simply go to the original router.
```

Comment:
Probably a better comment is, `Requests without bucket information should simply be processed in the current server.`